redlite/metric/_basic.py

```diff
@@ -49,56 +49,40 @@ def __engine(self, expected: str, actual: str) -> float:
             strip_punct=self.ignore_punct,
             normalize_whitespace=self.normalize_whitespace,
         )
+        return self.match(expected, actual)
 
-        if actual.startswith(expected):
-            return 1.0
-        return 0.0
+    def match(self, expected: str, actual: str) -> float:
```

Comment:
Rename to  `_match` , otherwise it is being picked up and documented by autodocs.